src/components/Cart/User/User.vue

```diff
@@ -1,34 +1,53 @@
 <template>
-  <div class="np-cart__user">
+  <div v-if="username !== ''" class="np-cart__user">
     <img :src="userAvatar" alt="avatar" class="np-cart__user__avatar" />
 
     <div class="np-cart__user__wrapper">
       <h2 class="np-cart__user__wrapper__name">{{ username }}</h2>
-      <a class="np-cart__user__wrapper__logout">{{ t("signOut") }}</a>
+      <a @click.prevent="signOut" class="np-cart__user__wrapper__logout">{{
+        t("signOut")
+      }}</a>
+    </div>
+  </div>
+  <div v-else class="np-cart__user">
+    <div class="np-cart__user__wrapper">
+      <h2 class="np-cart__user__wrapper__name">{{ t("notSignIn") }}</h2>
+      <router-link
+        :to="{ name: 'SignIn' }"
```

Comment:
I wolud make it constant and use it here and in router.
That wolud prevent further errors like changing router `name` to different one. In this case your `router-link` will not work